README.md

```diff
@@ -1,3 +1,43 @@
-# project-atelier
+# Timpani Trading Co.
+A modular, client-facing retail portal comprised of Product Overview, Related Products, Question & Answer, and Review widgets.
 
-This is where we will create thorough setup instructions.
+# Table of Contents
+// Link to the different sections below
+
+# Description
```

Comment:
Thanks again for doing the ReadMe.  Is this still WIP or is this a wrong version of the readme?

Reply:
The description section and table of contents are what I am meaning about the WIP